pkg/awsclient/elb_helper.go

```diff
@@ -78,7 +86,8 @@ func (c *awsClient) SetLoadBalancerPublic(elbName string, listenerPort int64) er
 // removeListenersFromELB will remove the 6443/TCP -> 6443/TCP listener from
 // the specified ELB. This is useful when the "ext" ELB is to be no longer
 // publicly accessible
-func (c *awsClient) removeListenersFromELB(elbName string) error {
+func (c *AwsClient) removeListenersFromELB(elbName string) error {
```

Comment:
What is the difference between this function and SetLoadBalancerPrivate, the descriptions seem to be similar.

Reply:
In Go, things with a capital letter are "exported" for use elsewhere. Think public methods and private methods from OOP languages. It's a common pattern to have the "public version" call some "private" method to complete the work.

Reply:
oh yeah!!!